lib/index.ts

```diff
@@ -30,6 +39,7 @@ async function buildDepTree(targetFileRaw: string, lockFileRaw: string): Promise
 
   const depTree: PkgTree = {
     dependencies: {},
+    hasDevDependencies: false,
```

Comment:
Please use targetFile.devDependencies as this seems to just be metadata about the package.json